src/com/amplitude/api/AmplitudeClient.java

```diff
@@ -916,63 +945,38 @@ private long getLongvalue(String key, long defaultValue) {
      * @return the next sequence number
      */
     long getNextSequenceNumber() {
-        long sequenceNumber = getLongvalue(SEQUENCE_NUMBER_KEY, 0);
         sequenceNumber++;
         dbHelper.insertOrReplaceKeyLongValue(SEQUENCE_NUMBER_KEY, sequenceNumber);
         return sequenceNumber;
     }
 
     /**
-     * Internal method to get the last event time.
-     *
-     * @return the last event time
-     */
-    long getLastEventTime() {
```

Comment:
I feel like it's cleaner if we just remove them? The thing is like the calling methods will save the values to local variables that have the same name as the instance variables. Having duplicate variables + extra level of indirection is kind of messy